lib/common/common.go

```diff
@@ -35,6 +35,12 @@ var (
 )
 
 const (
+	// AppcDockerImageName is the unmodified name this image was originally
+	// referenced by for fetching, e.g. something like "nginx:tag" or
+	// "quay.io/user/image:latest" This is identical in most cases to
+	// 'registryurl/repository:tag' but will differ for the Dockerhub registry or
+	// if the tag was inferred as latest.
+	AppcDockerImageName     = "appc.io/docker/imagename"
```

Comment:
`rawname` seems less clear, but I agree it's confusing. What about `pulled-name`? Something else?